app/models/user.rb

```diff
@@ -1,5 +1,13 @@
+# app/models/user.rb
 class User < ApplicationRecord
   has_many :comments
-  has_many :posts
+  has_many :posts, foreign_key: 'author_id'
   has_many :likes
+
+  validates :name, presence: true
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
```

Comment:
- [ ] To meet the below requirement please add unit tests for this custom method created for the user model :+1: 
![mm](https://github.com/NoelLincoln/blog-app/assets/89055502/1a7065fb-a00b-41cf-b3e5-3f72e0e2a85e)